WrightTools/diagrams/WMEL.py

```diff
@@ -432,3 +460,193 @@ def plot(self, save_path=None, close=False, bbox_inches="tight", pad_inches=1):
+        )  # get grid spec to confine cascade
+        sfig = self.fig.add_subfigure(gridspec[x, y])  # create subfigure
+
+        bbox = self.subplots[x - 1][y].get_position()  # get grid box position for cascade
+        spec = {
+            "width_ratios": [1 + 0 ** (i % 2) for i in range(3 * number - 3)]
+        }  # define width ratios for subplots
+        casc_subplot = sfig.subplots(
+            1, 3 * number - 3, gridspec_kw=spec, subplot_kw={"position": bbox}
+        )  # make subplots
+
+        # empirical adjustment to subfigure bbox to align levels with other row plots
+        sfig.subplots_adjust(
+            left=bbox_adjust[0], bottom=bbox_adjust[2], right=bbox_adjust[1], top=bbox_adjust[3]
+        )
+
+        # plot the arrows between cascading processes as a sine arrow
+        for arrowindex, arrowplot in enumerate(casc_subplot.flatten()[:-1]):
+            if arrowindex % 2 == 1:
+                xout = np.linspace(0, 4, 100)
+                yout = [np.sin(x * 2 * np.pi) for x in xout]
+                arrowplot.plot(xout, yout, color="k", linewidth=transfer_arrow_linewidth)
+                arrowplot.set_ylim(-20, 20)  # sets the visible amplitude of the sine wave
+                arr_lbl_x, arr_lbl_y = transfer_arrow_offset[0], transfer_arrow_offset[1]
+                arrowplot.annotate(
+                    r"$\mathrm{\hbar}$$\mathrm{\omega}$",
+                    [xout[len(xout) // 4] - arr_lbl_x, yout[len(yout) // 2] + arr_lbl_y],
+                )  # adds hv label
+                arrowplot.arrow(
+                    xout[-2], yout[-2], 0.0001, 0, head_width=2, head_length=1, fc="k", ec="k"
+                )  # add arrowhead
+
+        # plot the cascade's processes' diagrams
+        for ind, plot in enumerate(casc_subplot.flatten()):
+            plot.axis("off")
+            if ind % 2 == 0:
+                if titles != [None]:
+                    subtitles = []
+                    for t in titles:
+                        subtitles.append(t)
+                        subtitles.append("")
+                elif titles == [None]:
+                    subtitles = [""] * (2 * number_of_interactions)
+                print(subtitles)
+                if ind == 0 and label_side == "left":
+                    casc_wmel = Subplot(
+                        plot,
+                        energies=self.energies,
+                        number_of_interactions=number_of_interactions,
+                        title=subtitles[ind],
+                        title_font_size=title_font_size,
+                        state_names=state_names,
+                        virtual=virtual,
+                        state_font_size=state_font_size,
+                        state_text_buffer=state_text_buffer,
+                        label_side=label_side,
+                    )  # Uses the Subplot class to make the each process diagram
+
+                if ind == len(subtitles) - 2 and label_side == "right":
```

Comment:
elif to avoid falling through to the else condition for the first in this if/elif chain

Reply:
This is what is being indicated by the bot, though it's a little hard to decipher and I was at first like "nah, bot, you got that wrong", but it is actually correct